README.md

```diff
@@ -11,14 +11,23 @@ Converts asm to pseudo-C code.
 
 Requires [rizin](https://github.com/rizinorg/rizin).
 
-# Install
+# Install system wide
 
 Follow the following steps to install jsdec
  - clone this repository
  - Run `cd p`
  - Run `meson -Djsc_folder=".." build` to create the build folder
  - Run `ninja -C build install` to build the shared library and to install it 
 
+
+# Install in home folder
+
+If you want to install in the HOME folder, you can call `rizin -H RZ_USER_PLUGINS` for the exact user plugin path
+
+ - Run `cd p`
+ - Run ``meson -Djsc_folder=".." -Drizin_plugdir=`rizin -H RZ_USER_PLUGINS` build``
+ - Run `ninja -C build install`
+
```

Comment:
it is actually enough to do `--prefix=~/.local` instead of setting rizin_plugdir.

Reply:
let me try